inclusive-design/wecount.inclusivedesign.ca
# feat: apply proper themes to the cards used in homepage (resolves #1700)

## Description

See #1700 for details.

## Steps to test

1. Go to the homepage and scroll down to where the cards are
2. Change the UIO themes
3. Verify the images on the cards are changing colours based on the UIO themes

## Related issues

Resolves #1700
